### ISSUE TITLE
Add Popularity and Release Date to Movie Cards

### ISSUE THREAD

**Issue by ruveydayilmaz** [ruveydayilmaz/recomo#9]
**Description**
Currently, movie cards display only the movie name and vote average. To provide users with more information and improve the overall user experience, this issue aims to enrich the movie card data by including additional details such as the movie's popularity and release date.

**Expected Behavior**
Movie cards should now display the movie's name, average points, popularity, and release date, offering users a better understanding of each movie's key details at a glance. The popularity score should be presented using icons, with each icon representing increments of 100,000 users, ensuring a more visually appealing and user-friendly display.

**Tasks**
1. Update the `MovieCard.jsx` component to include fields for displaying the movie's popularity and release date.
2. Access the required data (popularity and release date) from the movie object. For example, use `movie.popularity` and `movie.release_date`.
3. Implement rendering of the popularity and release date information within the movie card.
4. Ensure that the existing functionality for displaying the movie name and average vote remains intact.
5. Test the updated `MovieCard.jsx` component with various movies to verify that the new data is correctly retrieved and displayed on the movie cards.
6. Apply responsive design adjustments to ensure the movie cards look visually appealing on different screen sizes.
7. Implement a logic to simplify the popularity score, converting the lengthy number to icons, with each icon representing increments of 100,000 users.
8. Utilize the provided icon for displaying popularity increments.

```JSON
  {
    "adult": false,
    "backdrop_path": "/nGxUxi3PfXDRm7Vg95VBNgNM8yc.jpg",
    "genre_ids": [
      28,
      12,
      16,
      878
    ],
    "id": 569094,
    "original_language": "en",
    "original_title": "Spider-Man: Across the Spider-Verse",
    "overview": "After reuniting with Gwen Stacy, Brooklyn’s full-time, friendly neighborhood Spider-Man...",
    "popularity": 1586.317,
    "poster_path": "/8Vt6mWEReuy4Of61Lnj5Xj704m8.jpg",
    "release_date": "2023-05-31",
    "title": "Spider-Man: Across the Spider-Verse",
    "video": false,
    "vote_average": 8.6,
    "vote_count": 1987
  }
```
The provided JSON snippet represents a movie. To improve readability, we will simplify the popularity display using icons, with each icon representing 100,000 users. We will use the provided icon from [here](https://icons8.com/icon/FR7Wf3HTQISd/trending). If the icon's default color does not match the theme, we will use the color `lightblue`.

**Additional Context**
Make sure to validate the data retrieval process to prevent issues with missing or incorrect popularity and release date information.

![Opera Anlık Görüntü_2023-07-05_105858_recomo vercel app](https://github.com/ruveydayilmaz/recomo/assets/74985368/5504e696-1f65-45f7-a82b-faa3ce0d20c4)

**Comment by karenhernandeze** [comment]
Can I be assigned this one?

**Comment by ruveydayilmaz** [comment]
@karenhernandeze you've got this! happy coding!

**Comment by karenhernandeze** [comment]
> @karenhernandeze you've got this! happy coding!

From where is the data being pulled? It is not populating

**Comment by ruveydayilmaz** [comment]
Hi @karenhernandeze , sorry for not mentioning it earlier, but I wanted to let you know that the integration with [The Movie Database (TMDB)](https://www.themoviedb.org/) requires the addition of some environment parameters. I have included instructions on how to set up these parameters in the readme file.

**Comment by karenhernandeze** [comment]
yeah I saw that only that I do not know how to access those variables or how to generate them on my own. Could you assist me on getting those?

**Comment by ruveydayilmaz** [comment]
@karenhernandeze ofc! Go to this link [TheMovieDB API Keys](https://developer.themoviedb.org/reference/intro/authentication#api-key-quick-start) and get your API key. You have to register first, and then you can add it to your env file.

**Comment by karenhernandeze** [comment]
@ruveydayilmaz hi! So I have created my account and was able to get the API Key. There are two: API Key and API Read Access Token, I tried with both, for the variable 'VITE_DB_API_KEY', yet nothing is loading. Do I need to add too the Cloudinary variables? Or am I doing something wrong?

**Comment by ruveydayilmaz** [comment]
@karenhernandeze  Hmm, so you searched for a movie name, but nothing was listed? You don't have to add Cloudinary, as it is only used for downloading or sharing a card.
